com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicPointsRenderer.cs

```diff
@@ -83,31 +86,51 @@ internal class AlembicPointsRenderer : MonoBehaviour
         Vector3 m_scale, m_scaleOld;
 
 
-        public Mesh sharedMesh
+        /// <summary>
+        /// The mesh to be instanced for every point cloud.
+        /// </summary>
+        public Mesh InstancedMesh
         {
             get { return m_mesh; }
             set { m_mesh = value; }
         }
 
-        public Material[] GetSharedMaterials() { return m_materials; }
-        public void SetSharedMaterials(Material[] value) { m_materials = value; }
+        /// <summary>
+        /// An array of materials used for the rendering the instanced mesh. Only one material per sub-mesh will be used.
+        /// </summary>
+        public List<Material> Materials
+        {
+            get
+            {
+                var ret = new List<Material>(m_materials.Length);
+                foreach (var t in m_materials)
+                {
+                    ret[0] = t;
+                }
+                return ret;
+            }
+            set { m_materials = value.ToArray(); }
+        }
 
-        public Material motionVectorMaterial
+        /// <summary>
+        /// Material to be used for the motion vector computation.
+        /// </summary>
+        public Material MotionVectorMaterial
         {
             get { return m_motionVectorMaterial; }
             set { m_motionVectorMaterial = value; }
         }
 
 
-        public void Flush()
+        void Flush()
```

Comment:
Just out of curiosity, is there convention of: when to explicitly add the "public" specifier or not?